src/NLog/Targets/EventLogTarget.cs

```diff
@@ -260,8 +265,17 @@ protected override void Write(LogEventInfo logEvent)
                 category = Convert.ToInt16(this.Category.Render(logEvent), CultureInfo.InvariantCulture);
             }
 
+            long maxKilobytes = 0;
+
+            if(this.MaxKilobytes != null)
+            {
+                maxKilobytes = Convert.ToInt64(this.MaxKilobytes.Render(logEvent), CultureInfo.InvariantCulture);
+            }
+
             EventLog eventLog = GetEventLog(logEvent);
 
+            eventLog.MaximumKilobytes = maxKilobytes;
```

Comment:
I was wondering why the unit tests on my local enviroment did not fail?? Is there any way to simulate the unit tests on appveyor without creating a pull request??